app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -24,165 +24,98 @@ expander:
-* your salary meets the requirement for a Skilled Worker visa
+You do not need a Skilled Worker visa to teach in England if you are an Irish citizen, or if you have one of the following: 
 
+* settled or pre-settled status under the [EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
 
-### Salary requirement for a Skilled Worker visa
- 
-The salary you need to get a visa depends on if you're employed as a qualified or unqualified teacher.
+* [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
 
-The school you're applying to work in can explain if the role you're applying for is qualified or unqualified. Qualified roles require more experience or qualifications.
-
-The salary requirement also varies depending on the region you're applying to work in. You can check which region you'll be working in with the school employing you.
+* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 
 
-If you're employed as a qualified teacher and work in:
+* [Family visa](https://www.gov.uk/uk-family-visa) 
 
-* inner London, you must earn at least $nonuk_skilledworkervisateacherminfulltime_innerlondon$
-* outer London, you must earn at least $nonuk_skilledworkervisateacherminfulltime_outerlondon$
-* London fringe, you must earn at least $nonuk_skilledworkervisateacherminfulltime_londonfringe$
-* the rest of England, you must earn at least $nonuk_skilledworkervisateacherminfulltime_restofengland$
+* [Graduate visa](https://www.gov.uk/graduate-visa) 
 
-If you're employed as an unqualified teacher and work in:
+* [High Potential Individual visa (HPI)](https://www.gov.uk/high-potential-individual-visa)
 
-* inner London, you must earn at least $nonuk_skilledworkervisatraineeminfulltime_innerlondon$
-* outer London, you must earn at least $nonuk_skilledworkervisatraineeminfulltime_outerlondon$
-* London fringe, you must earn at least $nonuk_skilledworkervisatraineeminfulltime_londonfringe$
-* the rest of England, you must earn at least $nonuk_skilledworkervisatraineeminfulltime_restofengland$
+* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa) 
 
-### Working part time as a teacher
+* [UK Ancestry visa](https://www.gov.uk/ancestry-visa) 
 
-You can work part-time, but your salary must meet the threshold for a Skilled Worker visa. There are 2 requirements:
+* [Youth Mobility Scheme visa](https://www.gov.uk/youth-mobility) 
 
-* your part-time salary must be at least $nonuk_skilledworkervisateacherminparttime$
-* the full-time equivalent of your salary must meet the minimum set for your region and status (for qualified and unqualified teachers)
+If you need to apply for a Skilled Worker visa, you will need to pay visa fees. You can find out [how much applying for a Skilled Worker visa will cost](https://www.gov.uk/skilled-worker-visa/how-much-it-costs).
```

Comment:
I find this sentence a bit awkward - `how much applying` sounds like it is a **lot** of applying!

Worth rewording (but please improve my suggestion!)

```suggestion
If you need to apply for a Skilled Worker visa, you will need to pay visa fees. You can find out [how much it costs to apply for a Skilled Worker visa will cost](https://www.gov.uk/skilled-worker-visa/how-much-it-costs).
```